zh-cn/docs/source_code_guide/adaptive-extension.html

```diff
@@ -27,7 +27,7 @@
         
     	<span class="hljs-comment">// 1.从 URL 中获取 WheelMaker 名称</span>
         String wheelMakerName = url.getParameter(<span class="hljs-string">"Wheel.maker"</span>);
-        <span class="hljs-keyword">if</span> (name == <span class="hljs-keyword">null</span>) {
```

Comment:
Hi, Please do not modify the html file directly. Please find the corresponding error in the source_code_guide/adaptive-extension.md file and correct it. Thx.